easybuild/easyconfigs/f/flex/flex-2.6.4.eb

```diff
@@ -25,4 +25,6 @@ dependencies = [
     ('M4', '1.4.18'),
 ]
 
+preconfigopts = 'export CPPFLAGS="$CPPFLAGS -D_GNU_SOURCE" && '
```

Comment:
@zao Please add a comment above mentioning https://github.com/westes/flex/issues/241 where this issue is discussed, and which confirms that this is the proper fix.

Reply:
Fairly sure that I found the workaround independently by delving into headers, but I'll reference the upstream discussion.

Reply:
Wasn't trying to imply that you didn't come up with the fix yourself, but it's good to have "official" confirmation. ;)